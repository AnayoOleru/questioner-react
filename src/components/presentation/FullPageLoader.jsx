import React, { Fragment } from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faSpinner } from '@fortawesome/free-solid-svg-icons';

const FullPageLoader = () => (
  <Fragment>
    <div className='FullPageLoader'>
      <span>
        <FontAwesomeIcon icon={faSpinner} size='3x' spin />
      </span>
    </div>
  </Fragment>
);

export default FullPageLoader;
