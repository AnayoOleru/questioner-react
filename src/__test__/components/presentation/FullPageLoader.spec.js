import React from 'react';
import { shallow } from 'enzyme';
import FullPageLoader from '../../../components/presentation/FullPageLoader';

describe('Full Page Loader Component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<FullPageLoader />);
  });
  test('Full Page Loader component should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
