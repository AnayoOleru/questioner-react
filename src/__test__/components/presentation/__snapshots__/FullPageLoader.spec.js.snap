// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Full Page Loader Component Full Page Loader component should render 1`] = `ShallowWrapper {}`;
